docs/dataset-systmone.md

```diff
@@ -96,9 +96,9 @@ For those with access to the OpenSAFELY database, the latest database build time
 
 ## OpenSAFELY-TPP database coverage and metadata
 
-- [Most recent coverage of records in SystmOne and external datasets](https://github.com/opensafely/database-notebooks/blob/master/notebooks/database-builds.ipynb)
+- [Most recent coverage of records in SystmOne and external datasets](https://reports.opensafely.org/reports/opensafely-tpp-database-builds/)
 - [Historical coverage of records in SystmOne and external datasets](https://github.com/opensafely/database-notebooks/blob/master/notebooks/database-history.ipynb)
-- [OpenSAFELY-TPP database schema](https://github.com/opensafely/database-notebooks/blob/master/notebooks/database-schema.ipynb)
+- [OpenSAFELY-TPP database schema]https://reports.opensafely.org/reports/opensafely-tpp-database-schema/)
```

Comment:
Think the opening `(` of the link accidentally got deleted in the edit, perhaps.

Otherwise, all good :slightly_smiling_face:

Reply:
haha -- this is why we review!